sources/foxaholic.py

```diff
@@ -59,7 +59,8 @@ def read_novel_info(self):
         self.novel_id = soup.select_one('#manga-chapters-holder')['data-id']
         logger.info('Novel id: %s', self.novel_id)
 
-        response = self.submit_form(chapter_list_url, data='action=manga_get_chapters&manga=' + self.novel_id)
+        # response = self.submit_form(chapter_list_url, data='action=manga_get_chapters&manga=' + self.novel_id)
+        response = self.submit_form(self.novel_url + 'ajax/chapters/', data='')
```

Comment:
Since `novel_url` may not always ends with a `/`, it is better to handle such case. A possible solution:

```
response = self.submit_form(self.novel_url.strip('/') + '/ajax/chapters/', data='')
```

Reply:
Fixed in 0d84f85